openrtb_ext/bid_request_video.go

```diff
@@ -136,6 +136,15 @@ type BidRequestVideo struct {
 	// Description:
 	//   Contains the OpenRTB Regs object to be passed to OpenRTB request
 	Regs *openrtb.Regs `json:"regs,omitempty"`
+
+	// TODO double check naming with client side
```

Comment:
Just want to follow up on this TODO.

On client side, this flag is called `PrioritizeDeals`. Since on Prebid Server we won't be changing the order of the bids based on whether they have deals or not, I think it should be named slightly different than client side, but not sure if `SupportDeals` is the best. Let me know if there's a better option.